router.js

```diff
@@ -0,0 +1,70 @@
+var express 	= require('express');
+const fs 		= require('fs');
+var router 		= express.Router();
+const path 		= require("path");
+
+
+//top level response
+router.get('/', function (req, res) {
+	res.type('text/plain');
+ 	res.end('Welcome to the Very Fancy Note Keeper');
+});
+
+
+//get list of resources
+router.get('/notes', function (req, res) {
+	var inpath = './data';
+	var outfiles = [];
+	var fileString='';
+
+	fs.readdir(inpath, function (err, files) {
+		if (err) {
+			throw err;
```

Comment:
Don't `throw` here, it will only take down your server. Send a different response (500) with message.